.buildkite/pipeline.yml

```diff
@@ -17,6 +17,16 @@ steps:
       docker-compose:
         run: gestalt
   - wait
+  - name: ":yarn:"
```

Comment:
This will run on every PR - where I think you only want it to run on master. Try looking into branch targeting.

Reply:
Actually the greenkeeper bot PRs are done in separate branches, not master, so that won't work. I think this is fine because the program has a way of detecting whether it came from the bot and if not then it short circuits like this
![image](https://user-images.githubusercontent.com/7877010/44924584-9539d280-ad00-11e8-8e83-175386798c31.png)